crates/storage/db/src/utils.rs

```diff
@@ -16,6 +17,11 @@ pub(crate) fn default_page_size() -> usize {
     os_page_size.clamp(min_page_size, libmdbx_max_page_size)
 }
 
+#[cfg(not(feature = "page_size"))]
+pub(crate) fn default_page_size() -> usize {
+    0
```

Comment:
this is a bit dangerous

this is used as an option, so I think we want to make the function that is used to configure the page size return an option instead

https://github.com/paradigmxyz/reth/blob/fc4c037e60b623b81b296fe9242fa905ff36b89a/crates/storage/libmdbx-rs/src/environment.rs#L632-L637